src/editions.md

```diff
@@ -0,0 +1,39 @@
+# Editions
+
+Before we get into the details of the language itself, we need to cover the
+concept of an "edition."
+
+An "edition" of Rust consists of a particular year. There are two editions of
+Rust, currently:
+
+* Rust 2015
+* Rust 2018
+
+Conforming compilers are expected to:
+
+* Provide a flag allowing users to select an edition.
+* Default to the 2015 edition if no edition is selected.
+
+The reference follows this guideline; everything stated in this document is
+part of the 2015 edition unless specifically marked otherwise.
```

Comment:
This seems like a good policy for now, given how small the actual "made into an error" class of things is.